packages/react-ssr/package-lock.json

```
@@ -1,12 +1,12 @@
 {
   "name": "@cdssnc/gcds-components-react-ssr",
-  "version": "0.39.0-canary.0",
+  "version": "0.40.0-canary.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "@cdssnc/gcds-components-react-ssr",
-      "version": "0.39.0-canary.0",
+      "version": "0.40.0",
```

```suggestion
      "version": "0.40.0-canary.0",
```

One canary tag escaped haha